src/components/usa-step-indicator/content/usa-step-indicator~center.json

```diff
@@ -0,0 +1,33 @@
+{
+  "modifier": "usa-step-indicator--center",
+  "showLabels": true,
+  "centered": true,
```

Comment:
The showLabels and Centered controls don't appear to do anything

Reply:
I can't find anything that changes with the centered tag _but_ the template uses the showLabels tag to display `aria-current:"true"` on the current step. I believe this is needed for accessibility :) 

```
{% if step.status == "current" and showLabels %} aria-current="true"{% endif %}>
        {% if step.label %}
```
_from line 4 in `usa-step-indicator.twig`_

Reply:
fixed in commit [bc600d5](https://github.com/uswds/uswds/pull/4466/commits/bc600d5ffc0335739a1016b84842470d4731d490)